### ISSUE TITLE
Some Scratch 3.0 projects not running

### ISSUE THREAD

**Issue by minegamer5570** [nathan/phosphorus#1524]
Hello.

I attempted to use Phosphorus to speed up my Scratch 3.0 project to find it does not work. Can you please figure out why? Thanks.

Project [https://scratch.mit.edu/projects/290784048/](https://scratch.mit.edu/projects/290784048/)

http://scratch.mit.edu/projects/290784048
http://phosphorus.github.io/#290784048

```
Error: HTTP 404: 
    at XMLHttpRequest.xhr.onload (https://phosphorus.github.io/phosphorus.js:224:23)
```

**Comment by MagiFire12** [comment]
Maybe it has something to do with the fact that scratch changed into html? I have the same exact problem with it. like, i just put in a link but it says it doesnt exist. I am on chromebook, and projects just freeze at times and messes up the timing badly. lol, uhm, yeah... i think the html thing is the problem. i reported a similar thread on scratch itself and they said it has something to do of scratch switching from something to html.

**Comment by minegamer5570** [comment]
That does sound like the cause of the issue. I believe the reason as to why some projects still work is because there is a copy of the scratch 2.0 version still on scratch. So some projects on Phosphorus may not even be running the latest version? It may be trying to find the scratch 2.0 project on newer projects but failing because it no longer generates the 2.0 project file.

**Comment by MagiFire12** [comment]
Yeah. what you said lol. But the chromebook problem? do you have any idea why chromebook doesnt work yet?

**Comment by CoolAbhi1290** [comment]
The fact that Scratch 3.0 is not running is because the sb2 files are now sb3 files on the scratch server, and they are stored in <canvas>, JavaScript and HTML format unlike before which was just Flash plugin. So it will be pretty easy to fix, but the only problem will be that we will not be able to run the projects which have not been run in scratch 3.0. When you run a scratch 2.0 or 1.0 project in scratch 3.0, it changes the sb# to sb3, so it might be a hard task to overcome that obstacle in order to improve upon this project. Maybe that's why they developers are not fixing it...

**Comment by Kiddo9000** [comment]
that could possibly be the reason as to why we may not get a fix for this issue.
